packages/client/ui/react-ui/src/providers/CrossmintAuthProvider.tsx

```diff
@@ -25,26 +25,32 @@ export type CrossmintAuthProviderProps = {
     appearance?: UIConfig;
     children: ReactNode;
     loginMethods?: LoginMethod[];
+    refreshRoute?: string;
 };
 
 type AuthStatus = "logged-in" | "logged-out" | "in-progress";
 
-type AuthContextType = {
+export interface AuthContextType {
     crossmintAuth?: CrossmintAuth;
     login: () => void;
     logout: () => void;
     jwt?: string;
     user?: SDKExternalUser;
     status: AuthStatus;
     getUser: () => void;
-};
+}
 
-export const AuthContext = createContext<AuthContextType>({
+const defaultContextValue: AuthContextType = {
+    crossmintAuth: undefined,
     login: () => {},
     logout: () => {},
+    jwt: undefined,
+    user: undefined,
     status: "logged-out",
     getUser: () => {},
-});
+};
+
+export const AuthContext = createContext<AuthContextType>(defaultContextValue as AuthContextType);
```

Comment:
Why is the `as AuthContextType` needed here? The variable is typed already.

Reply:
This was a leftover attempt at fixing a TypeScript issue. The actual fix was passing default values for `jwt`, `user` etc as undefined